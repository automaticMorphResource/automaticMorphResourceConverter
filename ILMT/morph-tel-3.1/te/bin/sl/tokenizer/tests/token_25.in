1802లో మేథ్యూ ప్లిండర్స్ అనే నావికుడొకరు మొట్టమొదటిసారిగా ఈ దీవిని కనుగొని కంగారూ ఐలాండ్ అనే పేరు పెట్టాడని చెబుతుంటారు. 
