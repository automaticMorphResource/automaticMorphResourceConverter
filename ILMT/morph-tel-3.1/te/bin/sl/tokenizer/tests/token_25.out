<Sentence id="1">
1	1802లో	unk
2	మేథ్యూ	unk
3	ప్లిండర్స్	unk
4	అనే	unk
5	నావికుడొకరు	unk
6	మొట్టమొదటిసారిగా	unk
7	ఈ	unk
8	దీవిని	unk
9	కనుగొని	unk
10	కంగారూ	unk
11	ఐలాండ్	unk
12	అనే	unk
13	పేరు	unk
14	పెట్టాడని	unk
15	చెబుతుంటారు	unk
16	.	unk
</Sentence>

